lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -80,4 +92,23 @@ export class BrowserAuthError extends AuthError {
     static createInteractionInProgressError(): BrowserAuthError {
         return new BrowserAuthError(BrowserAuthErrorMessage.interactionInProgress.code, BrowserAuthErrorMessage.interactionInProgress.desc);
     }
+
+    static createPopupWindowError(errDetail?: string): BrowserAuthError {
+        let errorMessage = BrowserAuthErrorMessage.popUpWindowError.desc;
+        if (errDetail && !StringUtils.isEmpty(errDetail)) {
+            errorMessage += ` Details: ${errDetail}`;
+        }
```

Comment:
Potential one-liner:

```js
const errorMessage = !StringUtils.isEmpty(errDetail) ? `${errorMessage} Details: ${errDetail}` : errorMessage;
```

(`StringUtils.isEmpty` already checks the truthiness of `errDetail`)

Reply:
+1

Reply:
done.